packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart

```diff
@@ -105,6 +107,17 @@ abstract class AuthenticatorPage {
     expect(expectCombinationNotFound, findsOneWidget);
   }
 
+  /// Then I see Invalid verification code
+  Future<void> expectInvalidVerificationCode() async {
+    if (Platform.isAndroid) {
```

Comment:
It surprised me that the messages were different on iOS and Android. Seems like this is a bug. Not specific to the authenticator though.

Reply:
Here is where the message on Android is coming from - https://github.com/aws-amplify/amplify-android/blob/main/core/src/main/java/com/amplifyframework/auth/AuthException.java#L317